apps/balancer-tools/src/app/apr/pool/(components)/PoolOverviewCards.tsx

```diff
@@ -21,20 +21,22 @@ async function AverageAPRCard({ poolId }: { poolId: string }) {
 }
 
 export default async function PoolOverviewCards({
-  roundId,
+  startAt,
+  endAt,
   poolId,
 }: {
-  roundId?: string;
+  startAt: Date;
+  endAt: Date;
   poolId: string;
 }) {
   const cardsDetails: {
     title: string;
     content: JSX.Element | string;
     tooltip?: string;
   }[] = [];
-  if (roundId) {
+  if (startAt || endAt) {
```

Comment:
per your definition, we'll always have startAt and endAt, so the `else` will never run. Is that expected?

Reply:
Used to be on `http://localhost:3000/apr/pool/ethereum/<poolId>` that would display the average return, but I don't think its useful anymore